e2e/nomostest/new.go

```diff
@@ -67,12 +67,15 @@ func newOptStruct(testName, tmpDir string, ntOptions ...ntopts.Opt) *ntopts.New
 			SyncSources: syncsource.Set{
 				// All tests default to having a RootSync named root-sync using
 				// a Git repository, unless otherwise specified.
-				DefaultRootSyncID: &syncsource.GitSyncSource{},
+				DefaultRootSyncID: &syncsource.GitSyncSource{
+					// RootSync root-sync defaults to using hierarchy.
+					// Use ntopts.SyncWithGitSource with ntopts.Unstructured to change.
+					SourceFormat: configsync.SourceFormatHierarchy,
```

Comment:
Doesn't need to be part of this refactor, but it would be nice to switch the default to unstructured. 99.9% of the time when I'm writing a new test I want to use unstructured mode

Reply:
Tempting. 

I think it was written this way to reflect the RootSync/RepoSync defaults, which are annoyingly different for reverse compatibility reasons.

It should probably just not specify the format by default, then it would use the actual server-side default. But at least this way it's more obvious to test writers that may not know the server-side defaults well.